modules/benchmark/R/clone_run.R

```diff
@@ -0,0 +1,115 @@
+##-----------------------------------------------------------------------------#
+##' Produce a "cloned" run from the settings of a completed run
+##' 
+##' @name clone_run 
+##' @title Clone Run
+##' @param settings settings or multisettings object 
+##' @param model model id 
+##' @param con database connection
+##' @export 
+##' 
+##' @author Betsy Cowdery 
+##' @importFrom dplyr tbl filter rename collect select 
+
+clone_run <- function(settings, model, con){
+  
```

Comment:
first thing needs to be a detection of Multisettings and recursive call to papply